test/unit/org/apache/cassandra/db/compaction/unified/ControllerTest.java

```diff
@@ -121,6 +121,50 @@ public void testValidateOptionsIntegers()
         testValidateOptions(true);
     }
 
+    @Test
+    public void testValidateOptionsInvalidTargetSSTableSize()
+    {
+        //Test 1: TARGET_SSTABLE_SIZE_OPTION = 12E899, the value reported in CASSANDRA-20398 
```

Comment:
why are there batches of tests rather than having an actual test case for each?  would be cleaner as it shows each issue rather than the first issue

Reply:
have split it. Thanks